src/promptflow-devkit/promptflow/_proxy/_python_inspector_proxy.py

```diff
@@ -21,6 +25,25 @@ def get_used_connection_names(
         return executable.get_connection_names(environment_variables_overrides=environment_variables_overrides)
 
     @classmethod
-    def is_flex_flow_entry(self, entry: str) -> bool:
+    def is_flex_flow_entry(cls, entry: str) -> bool:
         """Check if the flow is a flex flow entry."""
         return isinstance(entry, str) and re.match(FlowEntryRegex.Python, entry)
+
+    @classmethod
+    def get_entry_meta(
```

Comment:
@D-W- @YingChen1996 use this for direct meta generation? besides, if I understand correctly, we won't need to dump the json anymore?